### PR TITLE
[KOGITO-1162] Added services_image_version option

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-1162

Migrated some constant to `infrastructure` package as I don't think smoke package should rely on `cmd` and/or `controller` packages.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by radtriste** [comment]
smoke tests passed with images 0.8.0-rc1

**Comment by spolti** [comment]
great work, looks good to me, is it ready to merge?

**Comment by radtriste** [comment]
yes it is :)

File: pkg/framework/image.go [hunk 1]
```diff
@@ -42,6 +43,22 @@ func ConvertImageTagToImage(imageTag string) v1alpha1.Image {
 	return image
 }
 
+// ConvertImageToImageTag converts an Image into a plain string (domain/namespace/name:tag).
+func ConvertImageToImageTag(image v1alpha1.Image) string {
```
Comment (ricardozanini) [inline pkg/framework/image.go]: Framework package should be agnostic of our domain. I will think on something one I get back.

File: pkg/infrastructure/jobs_service.go [hunk 2]
```diff
@@ -22,6 +22,19 @@ import (
...
+	// DefaultJobsServiceImageNamespace is the default namespace for the Jobs Service image
+	DefaultJobsServiceImageNamespace = "kiegroup"
+	// DefaultJobsServiceImageName is the default image name for the Jobs Service image
+	DefaultJobsServiceImageName = "kogito-jobs-service"
+	// DefaultJobsServiceImageTag is the default image tag for the Jobs Service image
+	DefaultJobsServiceImageTag = "latest"
+	// DefaultJobsServiceImageFullTag is the default full tag name for the Jobs Service image
+	DefaultJobsServiceImageFullTag = DefaultJobsServiceImageDomain + "/" + DefaultJobsServiceImageNamespace + "/" + DefaultJobsServiceImageName + ":" + DefaultJobsServiceImageTag
+	// DefaultJobsServiceName is the default name for the Jobs Services instance service
+	DefaultJobsServiceName = DefaultJobsServiceImageName
```
Comment (ricardozanini) [inline pkg/infrastructure/jobs_service.go]: The name of the service should be jobs-service. The CR is already named ‘kogito-Jobs-service’. The same for the Data index, which I would change in the data index improvements jira. Also I’m not sure if moving this variables to this package makes much sense. I will rethink on data index refactoring. We will have a single package for creating services. A lot of code will go away. Thanks for the refactoring. Indeed the smoke should not depend on controller nor cmd. :)

Comment (radtriste) [reply pkg/infrastructure/jobs_service.go]: ok, we can discuss all of that when you get back :)

**Pull request merged**